tests/network/vmi_multus.go

```diff
@@ -566,6 +568,75 @@ var _ = SIGDescribe("[Serial]Multus", func() {
 				Expect(testErr.ErrStatus.Reason).To(BeEquivalentTo("Invalid"))
 			})
 		})
+
+		Context("MAC Spoof check with Linux bridge CNI plugin interface and custom MAC address.", func() {
+			It("Should allow outbound communication from VM under test - only if original MAC address is unchanged", func() {
+				const (
+					bridgeInterfaceName = "linux-bridge"
+					initialMacAddress   = "50:00:00:00:90:0e"
+					spoofedMacAddress   = "50:00:00:00:80:0a"
+				)
+
+				const (
+					vmUnderTestIPAddress = "10.2.1.1"
+					dummyVMIPAddress     = "10.2.1.2"
```

Comment:
The name "dummy" does not reflect that it is a reference IP for testing against.

Reply:
"dummy" was changed to "target"